javascript/src/main/pages/Todos/Todos.js

```diff
@@ -1,20 +1,22 @@
-import React from "react";
+import React, { useState } from "react";
 import useSWR from "swr";
 import { ListGroup } from "react-bootstrap";
+import { TodoFilters } from "./TodoFilters";
 import { TodoForm } from "./TodoForm";
 import { TodoItem } from "./TodoItem";
 import { TodoHeader } from "./TodoHeader";
 import { fetchWithToken } from "main/utils/fetch";
 import { useAuth0 } from "@auth0/auth0-react";
 import Loading from "main/components/Loading/Loading";
-import { sortTodos } from "../../utils/todoHelpers";
+import { sortTodos, filterTodo } from "../../utils/todoHelpers";
```

Comment:
Try to use non-relative paths when possible. In this case, I believe the following would work:
```suggestion
import { sortTodos, filterTodo } from "main/utils/todoHelpers";
```